tests/test_quadratic_model.py

```diff
@@ -685,6 +685,39 @@ def test_simple(self):
         self.assertEqual((-i*j - x).to_polystring(), '-x - i*j')
 
 
+class TestVartypeInfo(unittest.TestCase):
+    @parameterized.expand([(np.float32,), (np.float64,)])
```

Comment:
Looking at the tests, maybe the static method suggestion doesn't hold. Unless it wouldn't be much trouble to optionally pass a `dtype` arg to `vartype_info` itself, mainly for the case of the `Integer` vartypes.

Reply:
Yes, that was the reasoning. But now that you mention it, perhaps I should be thinking of this differently and just make it a function similar to the C++ level. Something like
```python
def vartype_info(dtype, vartype):
    ...
```
will consider